src/userdev/java/net/minecraftforge/gradle/userdev/UserDevPlugin.java

```diff
@@ -49,25 +52,29 @@
 import javax.annotation.Nonnull;
 import java.io.File;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.concurrent.TimeUnit;
-import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.toList;
 
 public class UserDevPlugin implements Plugin<Project> {
     private static String MINECRAFT = "minecraft";
     private static String DEOBF = "deobf";
+    public static String OBF = "__obfuscated"; //
 
     @Override
     public void apply(@Nonnull Project project) {
         Utils.checkJavaVersion();
 
         @SuppressWarnings("unused")
         final Logger logger = project.getLogger();
-        final UserDevExtension extension = project.getExtensions().create(UserDevExtension.class, UserDevExtension.EXTENSION_NAME, UserDevExtension.class, project);
+        final UserDevExtension extension = project.getExtensions().create(UserDevExtension.EXTENSION_NAME, UserDevExtension.class, project);
 
         if (project.getPluginManager().findPlugin("java") == null) {
             project.getPluginManager().apply("java");
         }
-        final File natives_folder = project.file("build/natives/");
+        final File nativesFolder = project.file("build/natives/");
```

Comment:
0.o

Reply:
Good 0.o or bad 0.o?

Reply:
Bad 0.o, why change names it's just midding your commit.

Reply:
Was named in snake case for some reason, so I decided it's a small enough change to just do it along the way. Want me to revert that?